src/users/containers/user-details/user-details.component.ts

```diff
@@ -70,6 +109,9 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
   }
 
   public ngOnDestroy(): void {
+    this.onDestory$.next();
```

Comment:
Please fix typo in "onDestroy$" 
Probably not a functional issue, but it's bound to trip someone up in future

Reply:
Fixed spelling for 'onDestroy' in commit:
https://github.com/hmcts/rpx-xui-manage-organisations/pull/1239/commits/58d56cf893eb33f411d8b82163bf30c343536915